src/common/dao/group/usergroup.go

```diff
@@ -84,6 +86,27 @@ func GetUserGroup(id int) (*models.UserGroup, error) {
 	return nil, nil
 }
 
+// GetGroupIDByGroupName ...
```

Comment:
This is trivial... but how about GetGroupIDByName?  The extra group is redundant...